### ISSUE TITLE
Error when compiling weatheralerts yaml

### ISSUE THREAD

**Issue by dbakez** [custom-components/weatheralerts#77]
When I check the yaml configuration I get the following error:

expected a dictionary for dictionary value @ data['packages']['sensor']

**Comment by IIAIronWolf** [comment]
I don't think this person is maintaining this anymore. I would try deleting and reinstalling.
